addons/website_sale_wishlist/models/product_wishlist.py

```diff
@@ -29,9 +29,9 @@ def current(self):
             return self
 
         if request.website.is_public_user():
-            wish = self.sudo().search([('id', 'in', request.session.get('wishlist_ids', []))])
+            wish = self.sudo().search([('id', 'in', request.session.get('wishlist_ids', [])), ('website_id', '=', request.website.id)])
```

Comment:
not sure to understand this one.
Your session is already website specific.

Reply:
Yes! You are right. changes altered. Thanks!